### DISCUSSION TITLE
After downgrading games when I open them it still opens the latest version.

### DISCUSSION THREAD

**Discussion by jacobnotfaze787** [SteamRE/DepotDownloader#533] [Q&A]
After I downgrade games when I open them it still opens the latest version this has happened with the two games I have tried and I have chosen the correct manifest id but it still just opens the latest version after deleting that file and replacing it with the depot with the same name as the file.

**Comment by psychonic** [comment]
That's usually a result of launching the downloaded files outside of Steam, and the game utilizing the [SteamAPI_RestartAppIfNecessary](https://partner.steamgames.com/doc/sdk/api#SteamAPI_RestartAppIfNecessary) function as a form of (very) light DRM.

You'll need to either copy the downloaded files to the original Steam game install directory, or find another way to launch that version over Steam.
